### ISSUE TITLE
Primary Deliverable - FossilSpecimen definition

### ISSUE THREAD

**Issue by Jegelewicz** [tdwg/material-sample#5]
## Current Definition

http://rs.tdwg.org/dwc/terms/FossilSpecimen

> A preserved specimen that is a fossil.

Please suggest changes/improvements in this issue.

## See also
https://github.com/tdwg/material-sample/blob/main/primary_deliverable/FossilSpecimen.markdown
http://www.ontobee.org/ontology/BCO?iri=http://rs.tdwg.org/dwc/terms/FossilSpecimen

**Comment by Jegelewicz** [comment]
The current definition and the BCO indicate that this is a subclass of [PreservedSpecimen](https://github.com/tdwg/material-sample/issues/3). Do we need subclasses or should [preparation](https://github.com/tdwg/material-sample/issues/7) be used instead?

FossilSpecimen = PreservedSpecimen +(preparation = fossil)

**Comment by tucotuco** [comment]
Additional related commentary in https://github.com/tdwg/material-sample/issues/3#issuecomment-903305155.

**Comment by smrgeoinfo** [comment]
FossilSpecimen (http://resource.isamples.org/vocabulary/specimentype/fossil): the remains of one or more organisms preserved in rock; includes whole body, body parts (usually bone or shell), and trace fossils. An organism or organism part becomes a fossil when it has undergone some fossilization process that generally entails physical and chemical changes akin to diagenesis in a sedimentary rock. Trace fossils are manifestations of biologic activity preserved in a rock body (typically sedimentary), without included preserved body parts. There are many processes that lead to fossilization, including permineralization, casts and molds, authigenic mineralization, replacement and recrystallization, adpression, carbonization, and bioimmuration (https://en.wikipedia.org/wiki/Fossil).

**Comment by tucotuco** [comment]
@smrgeoinfo This seems very clear for what a fossil is. Accepting it would "only" leave us with figuring out what a specimen is for a FossilSpecimen definition upgrade.

Should that first URL resolve to something other than a 404?

**Comment by smrgeoinfo** [comment]
Oops, thought we had set up the URIs to resolve. They don't yet.

**Comment by Jegelewicz** [comment]
> FossilSpecimen (http://resource.isamples.org/vocabulary/specimentype/fossil): the remains of one or more organisms preserved in rock; includes whole body, body parts (usually bone or shell), and trace fossils. An organism or organism part becomes a fossil when it has undergone some fossilization process that generally entails physical and chemical changes akin to diagenesis in a sedimentary rock. Trace fossils are manifestations of biologic activity preserved in a rock body (typically sedimentary), without included preserved body parts. There are many processes that lead to fossilization, including permineralization, casts and molds, authigenic mineralization, replacement and recrystallization, adpression, carbonization, and bioimmuration (https://en.wikipedia.org/wiki/Fossil).

I am not sure that everyone would agree with that definition. I think it would exclude [UTEP:ES:22-1609](https://arctos.database.museum/guid/UTEP:ES:22-1609)

would that then become PreservedSpecimen?

How about things that are "in process" like what I am looking at [here](https://the3dresearchlibrary.wordpress.com/2012/12/14/my-foray-into-fossils/)?

I think it is the first sentence of the definition that bothers me. Also, I am still thinking that "fossil" is relative and that modifiers to MaterialSample would let everyone use their preferred definition of "fossil" to find what they want: Things with an associated stratigraphy, things with an associated deposit date in MYA, things with "permineralizied" in preservation, etc.

**Comment by RogerBurkhalter** [comment]
My comment the @smrgeoinfo  
Member
My take is the @smrgeoinfo FossilSpecimen definition is too complex. Long ago I learned fossils were the "Remains of past life", a simple and inclusive definition. Some would say that is too inclusive and would include all of the biological collections in a museum but exclude most objects from a zoo. You could add "was deceased when collected" but that has issues as well. As a CM I have 70 million-year-old mollusks that are virtually unaltered and look as if they died yesterday. Marine and terrestrial palynomorphs are perhaps numerically the most common fossils in earth's entire history, but typically have minimal alteration after 100's of millions of years. @Jegelewicz is correct that the term "fossil" is relative and I think complicated definitions become exclusive. I know some archaeologists that consider excavated floral and faunal remains as fossils even if only a few hundred years old.

I don't think an alignment of FossilSpecimen with PreservedSpecimen is useful. Fossils are not typically something anthropogenically preserved, that is how that class is used, but are instead naturally preserved.

As far as "in process", I avoid the use of "sub-fossil". I do have many kg of material that are unprocessed, containing fossil specimens, that I record on my CMS as a "Sample" in much the same way as we are discussing MaterialSample. These "Sample", mainly from former curators or students, have a UUID and may become processed into specimens at some point.

**Comment by Jegelewicz** [comment]
> I don't think an alignment of FossilSpecimen with PreservedSpecimen is useful. Fossils are not typically something anthropogenically preserved, that is how that class is used, but are instead naturally preserved.

Does it really matter who preserved it? Does preservation imply that a human did it?

I think there may also be "naturally" preserved non-fossil specimens. How about a mummified animal found in the desert?

https://arctos.database.museum/guid/UTEP:Mamm:1750

Nobody "preserved" it, but it certainly is preserved and may or may not be a fossil depending on your definition.

**Comment by RogerBurkhalter** [comment]
I see your point, however, PreservedSpecimen is typically used for anthropogenic preservation as given in the DwC examples "A plant on an herbarium sheet. A cataloged lot of fish in a jar". Because these terms are usually aligned (best practice) with basisOfRecord (Other Deliverable - BasisOfRecord review #11) where, as a practical matter, FossilSpecimen is used by collections and researchers to include or exclude within data searches.  A search for "Canidae" AND "FossilSpecimen" yields a manageable dataset of only fossilized Canidae. The same is true for a search at the genus level where genera include both extant and fossil organisms. This efficiency is of huge importance in making paleontological specimens more accessible. Lumping FossilSpecimen with PreservedSpecimen makes the data more cumbersome and inefficient for both biologists and paleontologists. It is quite easy to include All by searching at whatever level you want and not filtering basisOfRecord.

**Comment by deepreef** [comment]
Certainly the fact that a physical `MaterialSample` item represents a fossilized impression of an organism (as opposed to directly preserved tissue from a recently living organism, or a still-living organism) is critical information that should be included within the spectrum of data exchanged using DwC terms.  But the question is: is `basisOfRecord` really the best way to capture this? 

Put another way, is the distinction between `MaterialSample`, `PreservedSpecimen`, `FossilSpecimen`, and `LivingSpecimen` more or less the same kind of distinction as that which divides `Location` from `Event` from `Taxon` from `Identification`, etc.?  The latter examples seem to me to be fundamentally different classes of things with very few (if any) shared properties; whereas the former examples seem to me to be slightly different kinds of things, which otherwise share largely the same set of properties.

In other words, I think we all feel that it's important to distinguish instances of `MaterialSample` that are fossils from those that are living or recently dead & preserved.  But it seems (to me at least) that treating them as different classes of things distinguished at the level of `basisOfRecord` is out of place.

**Comment by Jegelewicz** [comment]
> In other words, I think we all feel that it's important to distinguish instances of MaterialSample that are fossils from those that are living or recently dead & preserved. But it seems (to me at least) that treating them as different classes of things distinguished at the level of basisOfRecord is out of place.

Thank you - that is exactly what I have been trying to get at!

**Comment by Jegelewicz** [comment]
The Task Group has decided to deprecate this term and perhaps integrate it into controlled vocabulary for materialSampleType. This is a draft of the dwc issue to submit.

## Term change

* Submitter: TDWG MaterialSample Task Group
* Efficacy Justification (why is this change necessary?): Deprecate this term for consolidation of physical objects into the class MaterialSample rather than including specific classes for different kinds of physical objects which can be covered under materialSamplType
* Demand Justification (if the change is semantic in nature, name at least two organizations that independently need this term): 
* Stability Justification (what concerns are there that this might affect existing implementations?): 
* Implications for dwciri: namespace (does this change affect a dwciri term version)?: Yes

Current Term definition: https://dwc.tdwg.org/list/#dwc_FossilSpecimen

Proposed attributes of the new term version (Please put actual changes to be implemented in **bold** and ~strikethrough~):

* Term name (in lowerCamelCase for properties, UpperCamelCase for classes): ~FossilSpecimen~
* Organized in Class (e.g., Occurrence, Event, Location, Taxon): 
* Definition of the term (normative): ~A preserved specimen that is a fossil.~
* Usage comments (recommendations regarding content, etc., not normative): 
* Examples (not normative): ~A body fossil. A coprolite. A gastrolith. An ichnofossil. A piece of a petrified tree.~
* Refines (identifier of the broader term this term refines; normative): 
* Replaces (identifier of the existing term that would be deprecated and replaced by this term; normative): 
* ABCD 2.06 (XPATH of the equivalent term in ABCD or EFG; not normative):

**Comment by dagendresen** [comment]
Is there a way to say "replaced by" (on deprecated term level) -- pointing to the new controlled vocabulary term we plan to propose?

**Comment by baskaufs** [comment]
@dagendresen Yes, `dcterms:isReplacedBy`. See http://rs.tdwg.org/dwc/terms/individualID.rdf for an existing example, which also includes deprecation.

**Comment by Jegelewicz** [comment]
Based upon requested change in the MaterialSample definition change and taking the opportunity to make this definition more than an extended version of the term. I will add a revised definition for this term to the review package.

## Current Definition
FossilSpecimen - A preserved specimen that is a fossil.

## Proposed Definition
FossilSpecimen - A material entity that represents an entity of interest in whole or in part that is the preserved remains, impression, or trace of any once-living thing from a past geological age.

**Comment by Jegelewicz** [comment]
The Task Group participants today decided that changing this definition will raise issues that are beyond the scope of the Task Group. Ideally, the next step will be a Task Group to more fully develop Material terms with an extension.

Suggest removing this from the review package.
